We set all of the FTP clients to passive mode by default.  Or at least that's
the intention.

--- krb5-1.2.1/src/appl/gssftp/ftp/main.c.passive	Thu Jun 29 22:27:07 2000
+++ krb5-1.2.1/src/appl/gssftp/ftp/main.c	Wed Aug 16 13:15:08 2000
@@ -178,7 +178,7 @@
 	cpend = 0;	/* no pending replies */
 	proxy = 0;	/* proxy not active */
 #ifndef NO_PASSIVE_MODE
-	passivemode = 0; /* passive mode not active */
+	passivemode = 1; /* passive mode active by default */
 #endif
 	crflag = 1;	/* strip c.r. on ascii gets */
 	sendport = -1;	/* not using ports */
--- krb5-1.2.1/src/appl/gssftp/ftp/ftp.M.passive	Wed Aug 16 13:15:26 2000
+++ krb5-1.2.1/src/appl/gssftp/ftp/ftp.M	Wed Aug 16 13:17:19 2000
@@ -619,10 +619,11 @@
 will forward a copy of the user's Kerberos tickets to the remote host.
 .TP
 .B passive
-Toggle passive data transfer mode.  In passive mode, the client initiates
-the data connection by listening on the data port.  Passive mode may
-be necessary for operation from behind firewalls which do not permit
-incoming connections.
+Toggle passive data transfer mode off.  In passive mode, the client initiates
+the data connection by connecting to the data port.  Passive mode is
+often necessary for operation from behind firewalls which do not permit
+incoming connections, but may need to be disabled if you connect to an
+FTP server which does not support passive operation.
 .TP
 .B private
 Set the protection level on data transfers to ``private''.  Data
